rskj-core/src/main/java/co/rsk/rpc/modules/personal/PersonalModuleWalletEnabled.java

```diff
@@ -196,4 +196,10 @@ private long convertFromJsonHexToLong(String x) throws Exception {
             throw new Exception("Incorrect hex syntax");
         return Long.parseLong(x.substring(2), 16);
     }
+
+    private static String obfuscateOutput(String sensitiveInformation) {
```

Comment:
What can all this prevent or be helpful for?

The only thing that I see is that whoever sees the log can read the length of the password.

Reply:
I just removed this whole commit, it was just accessory